locale/en/docs/index.md

```diff
@@ -42,8 +42,9 @@ This documentation describes the built-in modules provided by Node.js. It does n
 
 ## ES6 features
 
-The [ES6 section](/en/docs/es6/) describes the three ES6 feature groups, and details which features are enabled by default in Node.js, alongside explanatory links. It also shows how to find which version of V8 shipped with a particular Node.js release.
+The [ES6 section](/en/docs/es6/) describes the three ES6 feature groups, and details which features are enabled by default in Node.js, alongside explanatory links. It also shows how to determine the version of  V8 which shipped with a particular Node.js release.
 
 ## Guides
 
-The [Guides section](/en/docs/guides/) has long-form, in-depth articles about Node.js technical features and capabilities.
+The [Guides section](/en/docs/guides/) has long-form, in-depth articles about various Node.js technical features and capabilities.
+。
```

Comment:
```suggestion
```

Reply:
- This is precisely what I don't understand. The original index.md showed up as having 50 lines when I went to edit, but in preview tabs, the same edit would only show up as having 49. So I did whatever necessary to return the line count to 50. Also before there was a double space whose existence was getting eaten eg. obscured by an automatic line-wrap in GH's mobile app, desktop browser, and any other vehicle of inspection I chose -simply because the underlying JavaScript or whatever is force-restyling all paragraph lines as left,justified,universe. It's 'I'm Little Emperor-of-this-Reich' nuances, just like that, which remind me why my input is not required in every situation.。